### PR TITLE
fixed bug in about page

### LINKED ISSUE #5220: Bug in About Page
## **Describe the bug**

- A clear and concise description of what the bug is.

## **Possible solution**

I guess Updated about page is loading after the old about page is displayed once

## **Screenshots**

aboutpagebug

I would like to work on this under LGMSOC

### PR DESCRIPTION
**Pull request by AB-san**
## Related Issue

- Info about Issue or bug

fix #5220 

#### Describe the changes you've made

A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.

## Screenshots

|        Original         |          Updated           |
| :---------------------: | :------------------------: |
| **original screenshot** | <b>updated screenshot </b> |

### CODE REVIEW

**Comment by AB-san** [comment]
@sairish2001  I have made some changes due to which the loading time of css is reduced to a much greater extent but there is still some delay.

**Comment by AB-san** [comment]
@sairish2001 If you have any other way to reduce the delay please let me know

**Review by sairish2001 — CHANGES_REQUESTED** [review]
```
<div class="collapse card rounded" id="about">
.....
</div>
```
there's a div in the `index.html` page, just remove it, the bug will be resolved no need of changing CSS in `about.html`

**Comment by AB-san** [comment]
> ```
> <div class="collapse card rounded" id="about">
> .....
> </div>
> ```
> 
> there's a div in the `index.html` page, just remove it, the bug will be resolved no need of changing CSS in `about.html`

@sairish2001  Please review this now , I hv made the requested changes

File: index.html [hunk 1]
```diff
@@ -16762,35 +16762,6 @@ <h4>For Polar Curves</h4>
...
-                        to make comments or
-                        even contributions then <a id="aboutContact" href="contact.html">contact us.</a></h5><br>
-
-                    <h5><strong>Developed By:</strong> Rajinderpal Singh <span class="badge badge-light" id="aboutBadge"
-                            style="cursor:pointer;"
-                            onclick="window.open('https://sairish2001.github.io')">Sairish</span>
-                    </h5>
-                </div>
-
             </div>
```
Comment (sairish2001) [inline index.html:16765]: You missed this </div> to delete

**Review by sairish2001 — CHANGES_REQUESTED** [review]

**Review by sairish2001 — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (AB-san) [reply index.html:16765]: Really sorry,
I have rectified it now

**Pull request merged**